### PR TITLE
T 23danko.lojanica

### PR DESCRIPTION
**Pull request by dankolojanica1**

### CODE REVIEW

File: endavaTraining/src/test/java/com/endavatraining/TestHeroesPage.java [hunk 1]
```diff
@@ -147,6 +147,34 @@ public void isHeroCountCorrect2(){
...
+        heroesPage.logout();
+
+    }
+
+    @DataProvider(name = "EditHeroLevelAboveTheLimit")
+    public Object[][] getDataFromEditHeroLevelAboveTheLimitDataProvider() {
+        return new Object[][]
+                {
+                        { "81" },
+                        { "-25"},
```
Comment (endavamarko) [inline endavaTraining/src/test/java/com/endavatraining/TestHeroesPage.java]: Ovo je ok,  inace ovde bi dobar test case bio uraditi neki boundary value analysis, sa granicnim vrednostima i ubaciti jednu dobru. Npr.  -1, 0, 10, 80, 81 i sl. Ali i ovako je u redu dok prati test case, moze commit
